google/oauth2/service_account.py

```diff
@@ -156,6 +159,9 @@ def __init__(
                 the JWT assertion used in the authorization grant.
             always_use_jwt_access (Optional[bool]): Whether self signed JWT should
                 be always used.
+            universe_domain (str): The universe domain. The default
+                universe domain is googleapis.com. If this value is the default
+                value, then self signed jwt will be used for token refresh.
```

Comment:
nit: Replace "If this value is the default value" with "in case of default value" or "for default".. otherwise reads a bitt awkward, too many "the value/default" in a row )

Reply:
done